src/lib/error/api-error.ts

```diff
@@ -102,52 +91,11 @@ const statusCode = (errorName: UnleashApiErrorName): number => {
     }
 };
 
-type ValidationErrorDescription = {
-    description: string;
+type UnleashErrorData = {
     message: string;
-    path?: string;
+    name: UnleashApiErrorName;
 };
 
-type UnleashErrorData =
-    | {
-          message: string;
-          documentationLink?: string;
-      } & (
-          | {
-                name: UnleashApiErrorNameWithoutExtraData;
-            }
-          | {
-                name: 'NoAccessError';
-                permission: string;
-            }
-          | {
-                name: 'AuthenticationRequired';
-                path: string;
-                type: string;
-            }
-          | {
-                name:
-                    | 'ValidationError'
-                    | 'BadDataError'
-                    | 'BadRequestError'
-                    | 'MinimumOneEnvironmentError'
-                    | 'InvalidTokenError';
-                details: [
-                    ValidationErrorDescription,
-                    ...ValidationErrorDescription[],
-                ];
-            }
-          | {
-                name: 'OwaspValidationError';
-                details: [
-                    {
-                        validationErrors: string[];
-                        message: string;
-                    },
-                ];
-            }
-      );
-
 export class UnleashError extends Error {
```

Comment:
Why is UnleashError in api-error.ts while all the other errors have their own files?

Reply:
Tests and source file have been renamed in 71cf90741920d08d545d4133880b979d0c61daa5.